packages/wrangler/src/user/commands.ts

```diff
@@ -121,3 +129,48 @@ export const whoamiCommand = createCommand({
 		});
 	},
 });
+
+export const authNamespace = createNamespace({
+	metadata: {
+		description: "🔐 Manage authentication",
+		owner: "Workers: Authoring and Testing",
+		status: "stable",
+	},
+});
+
+export const authTokenCommand = createCommand({
+	metadata: {
+		description:
+			"🔑 Retrieve the current OAuth token, refreshing it if necessary",
+		owner: "Workers: Authoring and Testing",
+		status: "stable",
+	},
+	behaviour: {
+		printBanner: false,
+		printConfigWarnings: false,
+	},
+	async handler(_, { config }) {
+		// Check if using global auth key/email (not supported for token retrieval)
+		const authFromEnv = getAuthFromEnv();
```

Comment:
I don't know the code but it looks like `getAuthFromEnv` assume that API key/email has higher priority over a token (all from env).

Is this document somewhere? (IMO the code would be a good place to document this).